common/src/test/scala/uk/ac/wellcome/test/fixtures/package.scala

```diff
@@ -4,26 +4,31 @@ import grizzled.slf4j.Logger
 
 import scala.util.Try
 
-package object fixtures extends FixtureComposers {
+package object fixtures
+  extends FixtureComposers
+    with ImplicitLogging {
 
   type TestWith[T, R] = T => R
 
   type Fixture[L, R] = TestWith[L, R] => R
 
-  def safeCleanup[L](resource: L)(f: L => Unit)(
-    implicit logger: Logger): Unit = {
+  def safeCleanup[L](resource: L)(f: L => Unit): Unit = {
```

Comment:
If you're not passing logger implicitly here (which I think it' nice to not do) you can just extend Logging directly instead of extending ImplicitLogging

Reply:
It means you can probably delete ImplicitLogging cause this was probably the reason for adding it in . the first place